### DISCUSSION TITLE
Andriod App?

### DISCUSSION THREAD

**Discussion by yamisins** [Eugeny/tabby#9427] [General]
@Eugeny would you be open to making an andriod version of tabby. I don't know if it sounds stupid but I find myself sometimes when not at pc using my phone and the closest one I can use is terminus. I prefer tabby on all my computers so it would be cool if their was an andriod version of tabby. Or even an ios app but I don't know how much work would go into that.

**Comment by Eugeny** [comment]
Sorry, not planned
